llvm/test/MC/EraVM/asm-parser/sp-changes.s

```diff
@@ -0,0 +1,30 @@
+; RUN: llvm-mc -triple eravm -filetype=asm %s -o - | FileCheck %s
+
+  .text
+foo:
+
+  add stack-=[r1], r1, r2
+  add stack-=[42], r1, r2
+  add stack-=[r1+42], r1, r2
+  add r0, r1, stack+=[r1]
+  add r0, r1, stack+=[42]
+  add r0, r1, stack+=[r1+42]
+  add stack-=[r1+42], r1, stack+=[r1+42]
+
+  nop r0, r0, stack+=[r1+42]
```

Comment:
There is no test for `nop` without any operands.

Reply:
Added `NOPrrr` instruction and `nop r0, r0, r0 -> nop` alias + test cases.

Reply:
I have double checked with the existing assembler and the syntax `nop stack+=[reg+imm]` as well as `nop stack -= [reg + imm]` are accepted; I have updated the spec -- please add the test for these cases too.